src/shared/codelens/codeLensUtils.ts

```diff
@@ -108,52 +99,33 @@ function makeLocalInvokeCodeLens(
     return new vscode.CodeLens(params.range, command)
 }
 
-function makeConfigureCodeLens({
-    document,
-    handlerName,
-    range,
-    workspaceFolder,
-    samTemplate,
-}: MakeConfigureCodeLensParams): vscode.CodeLens {
-    // Handler will be the fully-qualified name, so we also allow '.' & ':' & '/' despite it being forbidden in handler names.
-    if (/[^\w\-\.\:\/]/.test(handlerName)) {
-        throw new Error(`Invalid handler name: '${handlerName}'`)
-    }
-    const command = {
-        arguments: [workspaceFolder, handlerName, samTemplate],
-        command: 'aws.configureLambda',
-        title: localize('AWS.command.configureLambda', 'Configure'),
-    }
-
-    return new vscode.CodeLens(range, command)
-}
-
-async function getAssociatedSamTemplate(
-    documentUri: vscode.Uri,
-    workspaceFolderUri: vscode.Uri,
-    handlerName: string
-): Promise<vscode.Uri> {
-    const templates = detectLocalTemplates({
-        workspaceUris: [workspaceFolderUri],
-    })
-
-    for await (const template of templates) {
-        try {
-            // Throws if template does not contain a resource for this handler.
-            await CloudFormation.getResourceFromTemplate({
-                templatePath: template.fsPath,
-                handlerName,
-            })
-        } catch {
-            continue
-        }
-
-        // If there are multiple matching templates, use the first one.
-        return template
-    }
-
-    throw new Error(`Cannot find a SAM template associated with handler '${handlerName}' in: ${documentUri.fsPath}.`)
-}
+// TODO: Keep around if we want a linkage back to the template file?
```

Comment:
We can always fish it out of the git history. You're leaving "delete the code" endorphins on the table!

Reply:
Fine, but maybe I also like the warm fuzzies of having the old code around, ever thought of THAT?!